src/main/resources/application.properties

```diff
@@ -1,6 +1,6 @@
 spring.datasource.url=jdbc:mysql://localhost:3306/GreenCity?createDatabaseIfNotExist=true&useSSL=false&serverTimezone=Europe/Kiev
 spring.datasource.username=root
-spring.datasource.password=root
+spring.datasource.password=kate
```

Comment:
Maybe you shouldn't have to change the password?

Reply:
okey

Reply:
ok